processors/src/main/java/io/opentelemetry/contrib/eventbridge/EventToSpanEventBridge.java

```diff
@@ -76,7 +76,10 @@ public static EventToSpanEventBridge create() {
   @Override
   public void onEmit(Context context, ReadWriteLogRecord logRecord) {
     LogRecordData logRecordData = logRecord.toLogRecordData();
-    String eventName = logRecordData.getAttributes().get(EVENT_NAME);
+    if (!(logRecordData instanceof ExtendedLogRecordData)) {
+      return;
+    }
+    String eventName = ((ExtendedLogRecordData) logRecordData).getEventName();
```

Comment:
@LikeTheSalad @jack-berg @breedx-splk I didn't attempt to do any backcompat here, let me know if you'd like time to add that before the upcoming release or if it's ok to just include the breaking change in the release notes, thanks!

Reply:
I'm good with this change as-is and think it's fine to break events, since they're still so completely in flux.

Reply:
thanks, I've noted this in the proposed release notes: #1746